scripts/mkcloud

```diff
@@ -955,16 +955,19 @@ function onhost_reset_admin
 function crowbarupgrade_5plus
 {
     if iscloudver 6plus ; then
-        cloudsource=$upgrade_cloudsource onadmin prepare_cloudupgrade_nodes_repos_6_to_7
-        onadmin upgrade_ses_to_4
+        cloudsource=$upgrade_cloudsource onadmin prepare_cloudupgrade_nodes_repos
+        if [[ $cloudsource = develcloud6 ]]; then
+            onadmin upgrade_ses_to_4
+        fi
         onadmin zypper_patch_all
         onadmin upgrade_prechecks
-        if [[ $cloudsource = develcloud6 ]] || [[ $upgrade_cloudsource = develcloud7 ]]; then
+        if [[ $cloudsource = develcloud6 ]] || [[ $upgrade_cloudsource = develcloud7 ]] || \
+           [[ $cloudsource = develcloud7 ]] || [[ $upgrade_cloudsource = develcloud8 ]]; then
```

Comment:
how about simplifying the condition as 
```bash
[[ $cloudsource =~ devel ]] || [[ $upgrade_cloudsource =~ devel ]]
```

Reply:
Yep, good one

Reply:
the (less readable?) variant with further reduced duplication would be
```bash
[[ $cloudsource$upgrade_cloudsource =~ devel ]]
```